### DISCUSSION TITLE
How can I ensure that the provider installed in Breeze using `pip` is being used instead of the local version?

### DISCUSSION THREAD

**Discussion by shahar1** [apache/airflow#31325] [Q&A] [answered]
According to the [docs](https://github.com/apache/airflow/blob/main/dev/README_RELEASE_PROVIDER_PACKAGES.md#verify-by-contributors): 

```markdown
After you are in Breeze:

pip install apache-airflow-providers-<provider>==<VERSION>rc<X>

NOTE! You should `Ctrl-C` and restart the connections to restart airflow components and make sure new
provider packages is used.
```

However, when I run `pip install` and re-run `airflow triggerer` - it still utilizes the local (mounted) version.
How should I restart the connections properly? 

Initially I thought of `breeze restart`, but according to #31324 it's not relevant anymore :)

**Comment by potiuk** [comment, marked as answer]
If you look above- This is only when you the `--use-airflow-version` switch. In this case i will not mount the local sources, but it will use the version installed from PyPI for both providers you choose and airflow.  You can also do the same by choosing `--munt-sources remove` 

Maybe it would be a good idea if you update the docs and description to make it clearer. See Help for `--use-airflow-version`:

```
 --use-airflow-version                Use (reinstall at entry) Airflow version from PyPI. It can also be `none`, `wheel`, or `sdist`
     if Airflow should be removed, installed from wheel packages or sdist packages available 
    in dist folder respectively. Implies --mount-sources `remove`.                    │
│                                      (none | wheel | sdist | <airflow_version>)                      
```

**Reply by potiuk** [reply]
Actually the docs you refer to ONLY describe the case where you verify released version of providers - this is documentation of how to install and test with released version of airflow as well so I am not even sure if updates to documentation are needed - but feel free to put an emphasis on that aspect there.

**Reply by shahar1** [reply]
I'll try to come up with an optimization; contributors should be aware of what they should do when testing the providers :)

**Reply by potiuk** [reply]
Cool :)
